pkcs7/src/lib.rs

```diff
@@ -8,14 +8,20 @@
 #![forbid(unsafe_code, clippy::unwrap_used)]
 #![warn(missing_docs, rust_2018_idioms, unused_qualifications)]
 
+mod certificate_choices;
+mod cms_version;
 mod content_info;
 mod content_type;
+mod revocation_info_choices;
+mod signer_info;
```

Comment:
Shouldn't this module be public? Otherwise, how can we access the SignerInfo data?

Reply:
Yeah, these should probably be `pub` (or have specific re-exports).